apps/trading/components/sidebar/sidebar.tsx

```diff
@@ -17,7 +19,7 @@ import { useConnect, usePubKeys } from '@vegaprotocol/wallet-react';
 import { useGetCurrentRouteId } from '../../lib/hooks/use-get-current-route-id';
 import { useT } from '../../lib/use-t';
 import { ErrorBoundary } from '../error-boundary';
-import { useScreenDimensions } from '@vegaprotocol/react-helpers';
```

Comment:
SwapContainer is used in the sidebar and also on the swap page so it should not be contained in client-pages/swap but in components. Any components in a client-pages/mypage should be only used on that page anything shared should be in /components

Reply:
updated the structure